iothub/device/tests/IotHubDeviceClientTests.cs

```diff
@@ -1196,63 +1165,14 @@ public async Task IotHubDeviceClient_SendTelemetryAsync_WithoutExplicitOpenAsync
             await act.Should().ThrowAsync<InvalidOperationException>();
         }
 
-        [TestMethod]
-        public async Task IotHubDeviceClient_SendTelemetryAsync_BeforeExplicitOpenAsync_ThrowsInvalidOperationException()
-        {
-            // arrange
-            using var deviceClient = new IotHubDeviceClient(FakeConnectionString);
-
-            // act
-            Func<Task> act = async () =>
-            {
-                await deviceClient.SendTelemetryAsync(new TelemetryMessage());
-                await deviceClient.OpenAsync();
-            };
-
-            // assert
-            await act.Should().ThrowAsync<InvalidOperationException>();
-        }
-
-        [TestMethod]
-        public void IotHubDeviceClient_SendTelemetryAsync_AfterExplicitOpenAsync_DoesNotThrow()
```

Comment:
This wasn't an actual unit test (it tries to connect and fails), and we aren't even observing/awaiting the action. I moved it to E2E.